SPECS-EXTENDED/perl-Mail-SPF/perl-Mail-SPF.spec

```diff
@@ -0,0 +1,220 @@
+Summary:        Object-oriented implementation of Sender Policy Framework
+Name:           perl-Mail-SPF
+Version:        2.9.0
+Release:        23%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Mail-SPF
+Source0:        https://cpan.metacpan.org/authors/id/J/JM/JMEHNLE/mail-spf/Mail-SPF-v%{version}.tar.gz#/%{name}-v%{version}.tar.gz
+Patch0:         Mail-SPF-v2.8.0-POD.patch
+Patch1:         Mail-SPF-v2.8.0-testsuite.patch
+BuildRequires:  coreutils
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+BuildRequires:  perl(Error)
+BuildRequires:  perl(Module::Build)
+BuildRequires:  perl(Net::DNS) >= 0.62
+BuildRequires:  perl(Net::DNS::RR)
+BuildRequires:  perl(Net::DNS::Resolver)
+BuildRequires:  perl(Net::DNS::Resolver::Programmable) >= 0.003
+BuildRequires:  perl(NetAddr::IP) >= 4
+BuildRequires:  perl(Sys::Hostname)
+BuildRequires:  perl(Test::More)
+BuildRequires:  perl(Test::Pod) >= 1.00
+BuildRequires:  perl(URI) >= 1.13
+BuildRequires:  perl(URI::Escape)
+BuildRequires:  perl(base)
+BuildRequires:  perl(blib)
+BuildRequires:  perl(constant)
+BuildRequires:  perl(overload)
+BuildRequires:  perl(strict)
+BuildRequires:  perl(utf8)
+BuildRequires:  perl(version)
+BuildRequires:  perl(warnings)
+Requires:       perl(:MODULE_COMPAT_%(eval "$(perl -V:version)"; echo $version))
+Requires:       perl(Net::DNS) >= 0.62
+Requires:       perl(URI) >= 1.13
+Requires(post): %{_sbindir}/update-alternatives
+Requires(postun): %{_sbindir}/update-alternatives
+BuildArch:      noarch
+
+%description
+Mail::SPF is an object-oriented implementation of Sender Policy Framework
+(SPF). See http://www.openspf.org for more information about SPF.
+
+%prep
+%setup -q -n Mail-SPF-v%{version}
```

Comment:
Use autosetup instead.

Reply:
I left it alone for the comments of the patches.
What do you think, should we still use `autosetup` ?